net/openfortivpn/files/openfortivpn.init

```diff
@@ -0,0 +1,76 @@
+#!/bin/sh /etc/rc.common
+
+START=60
+USE_PROCD=1
+NAME=openfortivpn
+BIN=/usr/sbin/openfortivpn
+CONFIG=/var/etc/openfortivpn.config
+
+
+validate_certs_section() {
+	uci_load_validate openfortivpn certs "$1" "$2" \
+		'trusted_cert:string'
+}
+
+validate_openfortivpn_section() {
+	uci_load_validate openfortivpn service "$1" "$2" \
+		'enabled:uinteger' \
+		'host:string' \
+		'port:uinteger' \
+		'username:string' \
+		'password:string' \
+		'set_routes:uinteger' \
+		'set_dns:uinteger' \
+		'pppd_use_peerdns:uinteger'
+}
+
+setup_certs() {
+	[ "$2" = 0 ] || {
+		echo "validation failed"
+		return 1
+	}
+
+	[ -n "$trusted_cert" ] || return 0
+	echo "trusted-cert = $trusted_cert" >> $CONFIG
+}
+
+setup_config() {
+	[ "$2" = 0 ] || {
+		echo "validation failed"
+		return 1
+	}
+
+	[ "$enabled" -eq 0 ] && return 1
```

Comment:
@lucize maybe this?

Reply:
Yeah that's probably it. I had a similar problem with transmission when I did this there (in that case it didn't work right).

Shellcheck's quoting warnings are sometimes questionable. Probably because it doesn't have enough information about /etc/rc.common or the other OpenWrt scripts.

Reply:
it was just a typo enable vs enabled between config and init